src/Uno.UI/Controls/CommandBarRenderer.iOS.cs

```diff
@@ -113,8 +113,9 @@ protected override void Render()
 
 			// CommandBarExtensions.BackButtonIcon
 			var backButtonIcon = Element.GetValue(BackButtonIconProperty) is BitmapIcon bitmapIcon
-				? UIImageHelper.FromUri(bitmapIcon.UriSource)
+				? UIImageHelper.FromUri(bitmapIcon.UriSource)?.ImageWithRenderingMode(UIImageRenderingMode.AlwaysOriginal)
```

Comment:
Do you understand why this is needed? Is it something that changed between iOS versions?

Reply:
@davidjohnoliver  I believe this is something that changed in iOS11 but I have been looking through the docs and blogs and haven't been able to find it.